src/components/organisms/Form/FormRadioGroupField/FormRadioGroupField.md

```diff
@@ -0,0 +1,31 @@
+### Summary
+
+`<Form.RadioGroupField>` is just a small wrapper around the [`<RadioGroupField />`](#/Components/Molecules/RadioGroupField) component. It is meant to be used **ONLY** inside [`<Form>`](#/Organisms/Form/Form) along with other [Form](#/Organisms/Form) components. Thanks to the form data passed down through context, `<Form.RadioGroupField>` already handles value change.
+
+### Example
+
+```js
+import { Form } from '@zopauk/react-components';
+
+<Form initialValues={{ employment: '' }} onSubmit={values => alert(JSON.stringify(values))}>
+  <Form.Form>
+    <div style={{ marginBottom: '16px' }}>
```

Comment:
Can we just put this inline style in `Form` to avoid adding an extra `div`? _This is a generic question regarding if `Form` is extendable_

Reply:
In most case, props are spread over the `<Form />` and its components so you can do `styled(<FormComponent />)`.  However, there are some more complex components such as `FormRadioGroupField` where I'm not sure if `className` should be passed to the wrapper or some other internal element.